clang/test/dpct/api_mapping_cases/Math/__hfma.cu

```diff
@@ -0,0 +1,11 @@
+// UNSUPPORTED: cuda-8.0, cuda-9.0, cuda-9.1, cuda-9.2, cuda-10.0, cuda-10.1, cuda-10.2, cuda-11.0, cuda-11.1, cuda-11.2, cuda-11.3, cuda-11.4, cuda-11.5
```

Comment:
add readme to add new test cases in clang/test/dpct/api_mapping_cases

Reply:
added